pkgs/development/compilers/ghc/8.10.4.nix

```diff
@@ -158,6 +160,11 @@ stdenv.mkDerivation (rec {
     # Fix documentation configuration which causes a syntax error with sphinx 4.*
     # See https://gitlab.haskell.org/ghc/ghc/-/issues/19962, remove at 8.10.6.
     ./sphinx-4-configuration.patch
+
+    (fetchpatch {
```

Comment:
Can you add an annotation in a comment above this on what it does / why it is needed and whether it has been upstreamed.

Is this change included in Cabal 3.4? If yes, I guess GHC 8.8.4 will also need this patch, if no practically all GHCs we have in nixpkgs.

Reply:
Huh, didn't mean to add this patch unconditionally, as non-cross has been working fine. 

> Is this change included in Cabal 3.4

Issue's still open, doesn't look like it's been fixed. Patch might be upstreamable though?

Pushed new version including it for cross only, and with a link to upstream issue.